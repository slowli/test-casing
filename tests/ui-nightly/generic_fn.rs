use test_casing::test_casing;

#[test_casing(2, ["test", "this"])]
fn tested_function<S: Into<String>>(_arg: S) {
    // Does nothing
}

fn main() {}
