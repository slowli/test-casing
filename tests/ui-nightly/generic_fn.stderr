error: generic tested functions are not supported
 --> tests/ui-nightly/generic_fn.rs:4:20
  |
4 | fn tested_function<S: Into<String>>(_arg: S) {
  |                    ^^^^^^^^^^^^^^^
